gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/kafka/writer/KafkaWriterHelper.java

```diff
@@ -90,4 +94,32 @@ public static Object getKafkaProducer(Properties props) {
       throw Throwables.propagate(e);
     }
   }
+
+  public static <K, D> K getKey(D record, KafkaWriterCommonConfig commonConfig)
```

Comment:
How come some places it is <K, D> or <D> and others it is <K, V>?

Reply:
will standardize